### DISCUSSION TITLE
Secure Cookie 사용에 관하여

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#311] [05. General]
현재 토큰의 경우 Cookie 로 관리하고 있습니다.
그러나 브라우저에서 Cookie를 확인할 수 있는데요,
HTTP Only Cookie의 경우, 자바스크립트 코드로(개발환경에서도..) 쿠키를 접근할 수 없기 때문에
쿠키를 직접 클라이언트에서 이용하는 저희 로직에서는 사용할 수 없는데,
Secure Cookie 같은 경우는 사용할 수는 있습니다.
그러나 Secure Cookie 의 경우, **쿠키를 전송할 때** HTTPS 에서만 쿠키를 전송할 수 있는 옵션이라서
현재 저희의 로직에서는 설정하나마나 동작 상 다를 게 없습니다. 
(왜냐하면 서버와 쿠키를 직접 주고 받지 않거든요)

따라서 쿠키에 대해서 Secure 설정을 하지 않아도 괜찮을까요?

**Comment by gitul0515** [comment]
same-site 설정
